R/manual_layer_node.R

```diff
@@ -357,7 +360,7 @@ Node <- R6::R6Class(
 ##'
 ##' @family {manual-layer functions}
 ##' @export
-compute <- function(delayed_function, namespace=NULL, timeout_seconds=NULL, verbose=FALSE, force_all_local=FALSE) 0
+compute <- function(delayed_function, namespace=NULL, timeout_seconds=NULL, verbose=FALSE, force_all_local=FALSE) {}
```

Comment:
This will be rebased atop https://github.com/TileDB-Inc/TileDB-Cloud-R/pull/72 once https://github.com/TileDB-Inc/TileDB-Cloud-R/pull/72 is accepted and merged.